static/app/views/issueList/utils/newTabContext.tsx

```diff
@@ -1,34 +1,38 @@
 import {createContext, useState} from 'react';
 
+export type NewView = {
+  label: string;
+  query: string;
+  saveQueryToView: boolean;
```

Comment:
Could use a comment here to explain what `saveQueryToView` means, maybe I'm dumb but I'd think that the query should always be saved to the view

Reply:
Good idea I'll add it in! For reference, it refers to whether or not the query is _persisted_ to the query or not. Basically, if `saveQueryToView=false`, then the unsaved changes indicator will be present in the tab. If it's true, then the tab will be persisted with the query. In the UI, clicking just the query's row will have `saveQueryToView=false`, and clicking the "Save as View" button will have `saveQueryToView=true`. This behavior was also explicitly specified in the figma mockups.